src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/CNNScoreVariants.java

```diff
@@ -319,10 +319,14 @@ private void transferReadsToPythonViaFifo(final VariantContext variant, final Re
         }
         Iterator<GATKRead> readIt = readsContext.iterator();
         if (!readIt.hasNext()) {
-            logger.warn("No reads at contig:" + variant.getContig() + "site:" + String.valueOf(variant.getStart()));
+            logger.warn("No reads at contig:" + variant.getContig() + " site:" + String.valueOf(variant.getStart()));
         }
         while (readIt.hasNext()) {
-            sb.append(GATKReadToString(readIt.next()));
+            GATKRead r  = readIt.next();
+            if (r.getReadGroup().toLowerCase().contains("artificial")){
```

Comment:
@lucidtronix Whats the reason for this change ? It seems to be casting a pretty wide net. HC-generated artificial read groups ?

Reply:
Yeah exactly, we've always excluded them from the read tensor.  But I agree, I will add an argument to toggle this.